README.md

```diff
@@ -15,11 +15,11 @@ git remote set-head origin -a
 
 ## Summary
 
-This is a collection of support libraries for Java applications (Java 8 and above) that serves three main purposes: 
+This is a collection of support libraries for Java applications (Java 11 and above) that serves three main purposes: 
 
 1. Provide means to emit *structured application log messages* 
 2. Instrument parts of your application stack to *collect request metrics* 
-3. Allow production of *custom metrics*.
+3. Allow auto-configuration of OpenTelemetry exporters.
```

Comment:
really only auto configuration?

Reply:
Almost. There is specific documentation on the component.